src/main/java/pl/edu/pw/pik/pikactivitytrackerserver/controller/WebsiteController.java

```diff
@@ -31,6 +33,17 @@
         return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
     }
 
+    @GetMapping("/getAll/{id}")
+    public ResponseEntity<List<Website>> getUsersWebsites(@PathVariable Integer id)
+    {
+        List<Website> lW = websiteService.getWebsitesByUserId(id);
+
+        if(lW != null)
+            return new ResponseEntity<>(lW, HttpStatus.OK);
+
+        return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
+    }
+
     @PostMapping("/addWebsite")
```

Comment:
w add webside gdy próbujemy dodać stronę która już istnieje apka zwraca Internal Server Error prawdopodobnie nie sprawdzamy czy taka strona juz istnieje i próbujemy dodać ją do bazy a baza wyrzuca value violates unique constraint. Nie wiem czy tak powinno być, może lepiej sprawdzać czy taka strona jest i jak nie to rzucać błędem 400